src/js/uart.js

```diff
@@ -0,0 +1,95 @@
+/* Copyright 2016 Samsung Electronics Co., Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+var EventEmitter = require('events').EventEmitter;
+var util = require('util');
+var assert  = require('assert');
+
+var uart = process.binding(process.binding.uart);
+
+//  VALIDATION ARRAYS
+var BAUDRATE = [0, 50, 75, 110, 134, 150, 200, 300, 600, 1200, 1800, 2400
+                , 4800, 9600, 19200, 38400, 57600, 115200, 230400];
```

Comment:
`BAUDRATE` is unused variable. How about using this to check for valid baudrate like `DATABITS`?